xml/System.Composition.Hosting/ContainerConfiguration.xml

```diff
@@ -268,11 +268,11 @@
         <Parameter Name="conventions" Type="System.Composition.Convention.AttributedModelProvider" />
       </Parameters>
       <Docs>
-        <typeparam name="TPart">To be added.</typeparam>
-        <param name="conventions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TPart">The type to add.</typeparam>
+        <param name="conventions">The conventions to use for the part type.</param>
+        <summary>Adds the specified generic type to the container.</summary>
```

Comment:
To disambiguate this from the other overload, "Adds the specified type to the container using the specified conventions."